nossomercadolivre-api/src/main/java/com/github/nossomercadolivre/UserDTO.java

```diff
@@ -1,68 +1,40 @@
 package com.github.nossomercadolivre;
 
-import javax.validation.constraints.*;
-import java.util.List;
-
-import static java.util.stream.Collectors.toList;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
 
 public class UserDTO {
     private Long id;
 
     @NotNull
     @NotEmpty
     @Email
-    @Size(min = 5, max = 100)
-    private String login;
+    @Size(max = 100)
+    public String email;
 
     @NotEmpty
     @NotNull
     @Size(min = 6, max = 200)
-    private String password;
+    public String password;
 
     @Deprecated
     /**
      * For frameworks, don't use it
      */
-    protected UserDTO() {
+    private UserDTO() {
     }
 
     public UserDTO(User user) {
```

Comment:
Aqui eu continuo achando que deveríamos uma classe só para a saída. Se você considera o risco válido, ok :).

Reply:
Pensando em futuro, concordo que pode haver a necessídade, porém, pro momento acredito que deveria deixar essa decisão pro ultimo momento responsável. Em outras palavras, prefiro deixar pra fazer isso quando for necessário, se não vou ter uma entidade que representa o banco, e dois DTOs (um de entrada e um de saída), me parece muito verboso pro momento.

Reply:
fechado :).